test/functional/feature_block.py

```diff
@@ -548,7 +548,7 @@ def run_test(self):
         tx.rehash()
         new_txs.append(tx)
         self.update_block(40, new_txs)
-        self.send_blocks([b40], success=False, reject_reason='bad-blk-sigops', reconnect=True)
+        self.send_blocks([b40], success=False, reject_reason='bad-blk-sigops', reconnect=True, timeout=200)
```

Comment:
You might just bump the default in `def send_blocks(...`.

However, feature_block.py might be the hardest to get right, as it includes a 1080 block reorg. Not sure how long that will take, but you might want to `export TEST_RUNNER_EXTRA="--exclude feature_block` for now.